frontend/src/pages/CoursesList/styled.ts

```diff
@@ -75,3 +76,33 @@ export const MobileSearchWrapper = styled('div')({
     display: 'none',
   },
 });
+
+export const AddButtonWrapper = styled('div')({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'end',
+  margin: '20px 0',
+  [theme.breakpoints.down(950)]: {
+    display: 'none',
+  },
+});
+
+export const AddButton = styled(Button)({
+  width: '85px',
```

Comment:
you should use one of the button variants from theme, for example variant `mediumContained` variant already has fontFamily, font size 16px, line-height 22px, letter-spacing -0.4px, textAlign: 'center', borderRadius: '4px',  boxShadow: 'none' etc, so you can remove at least 7 extra lines using custom button variant

Reply:
`<Button variant="mediumContained">`